helm/scaphandre/values.yaml

```diff
@@ -18,6 +18,8 @@ scaphandre:
     containers:
 #  rustBacktrace: '1'
 
+# Gives full host access, bypassing container isolation. true or false
+privileged: false
```

Comment:
@CalmaIndustry If the chart does not work with `privileged: false` (cf https://github.com/hubblo-org/scaphandre/issues/351), then why is it the default value?

Maybe if should be `true` by default, or some comment/documentation should explain why it is `false` by default but must be set to `true` for the user to acknowledge the implication of this setting.

Reply:
I don’t encounter the issue when running Scaphandre on a Kubernetes server hosted on a VM, but it only occurs on a Bare Metal Server. Therefore, for security reasons, I want to set it to false by default, but provide the option to enable it if the problem arises.

Reply:
> Therefore, for security reasons, I want to set it to false by default, but provide the option to enable it if the problem arises.

@CalmaIndustry Thank you for explaining it. That makes sense.

Still, IMO it should be documented *why* this option exists, otherwise people are going to keep opening issues like #351.

Reply:
Should i add the explanation in the README or directly on the values file ? Sorry it is my first pr I'am a bit confused about best practice

Reply:
@CalmaIndustry IMO you should update the docs: https://github.com/hubblo-org/scaphandre/blob/e09dfcc7abd28291e783643e366393325fad3f6d/docs_src/tutorials/kubernetes.md